### DISCUSSION TITLE
What is default username and password for traefik LXC container?

### DISCUSSION THREAD

**Discussion by kevdogg** [tteck/Proxmox#3253] [Help Needed] [answered]
What is default username and password for traefik LXC container?

**Comment by tteck** [comment]
There's no "default username and password"

It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.

**Reply by kevdogg** [reply]
Hey thanks for reply.  I managed to install the container days ago and restarted the container today and popped open a console window.  It's asking for a username/password to log in.  If you can't log into container, how are you supposed to edit configuration files?

**Reply by tteck** [reply, marked as answer]
If it's asking for credentials that you didn't provide in the Advanced settings, the installation script did not complete successfully.
